packages/plugin-dlx/sources/commands/dlx.ts

```diff
@@ -43,7 +43,29 @@ export default class DlxCommand extends BaseCommand {
 
       await xfs.writeFilePromise(ppath.join(tmpDir, toFilename(`package.json`)), `{}\n`);
       await xfs.writeFilePromise(ppath.join(tmpDir, toFilename(`yarn.lock`)), ``);
-      await xfs.writeFilePromise(ppath.join(tmpDir, toFilename(`.yarnrc.yml`)), `enableGlobalCache: true\n`);
+
+      const projectCwd = (await Configuration.findProjectCwd(this.context.cwd, Filename.lockfile))!;
+      const localConfigPath = ppath.join(projectCwd, toFilename(`.yarnrc.yml`));
+      const tmpDirConfigPath = ppath.join(tmpDir, toFilename(`.yarnrc.yml`));
+      if (await xfs.existsPromise(localConfigPath)) {
+        await xfs.copyFilePromise(localConfigPath, tmpDirConfigPath);
+        await Configuration.updateConfiguration(tmpDir, (current: any) => {
+          if (typeof current.plugins === `undefined`)
+            return {enableGlobalCache: true};
+
+          return {
+            enableGlobalCache: true,
+            plugins: current.plugins.map((plugin: any) => {
+              if (typeof plugin === `string`)
+                return ppath.resolve(projectCwd, plugin as PortablePath);
+
+              return {spec: plugin.spec, path: ppath.resolve(projectCwd, plugin.path as PortablePath)};
```

Comment:
The path is a native path (`c:\foo` on Windows), since it's user-facing.

```suggestion
              if (typeof plugin === `string`)
                return npath.resolve(projectCwd, plugin);

              return {spec: plugin.spec, path: npath.resolve(projectCwd, plugin.path)};
```